### PR TITLE
Adds a react() function to Conversations

### PR DESCRIPTION
**Pull request by RafaelCosman**
```
convo.react("+1");
```

### CODE REVIEW

File: lib/CoreBot.js [hunk 1]
```diff
@@ -126,6 +126,18 @@ function Botkit(configuration) {
         this.say = function(message) {
             this.addMessage(message);
         };
+        
+        this.react = function(reaction) {
```
Comment (anonrig) [inline lib/CoreBot.js]: Can you add a JSDoc?

File: lib/CoreBot.js [hunk 2]
```diff
@@ -126,6 +126,18 @@ function Botkit(configuration) {
...
+            bot.api.reactions.add({
+                timestamp: this.transcript[this.transcript.length-1].ts,
+                channel: this.source_message.channel,
+                name: reaction
+            },function(err, res) {
```
Comment (anonrig) [inline lib/CoreBot.js]: Did you use the linter? 

According to linter there should be an empty space between `,` and `function`

Please test your code using `npm test`

**Comment by RafaelCosman** [comment]
@anonrig Thanks! 

Also, I think that while this approach can work, it's not ultimately the right way to do it. Nothing in `CoreBot.js` should be calling the Slack API. Those calls should be in one of the slack-specific files.

**Comment by RafaelCosman** [comment]
Now passes linter. `5 passing (1s)`. Still would not recommend merge because this code should really be abstracted out of `CoreBot.js`.

**Comment by anonrig** [comment]
What about making the reaction an `enum`? @RafaelCosman

**Comment by karlbright** [comment]
+1 to moving this out of Corebot before it's merged and the idea of making this an `enum` is pretty sweet.

**Comment by RafaelCosman** [comment]
@anonrig and @karlbright, great suggestions! I have another PR that I'm focusing on right now so I'm going to temporarily mark this PR with #DONT_MERGE. I'll leave it open so that others who want to work on this or just want to use reactions can use this temporarily until we get a complete fix.

**Pull request closed without merging**